api/controllers/VisitsController.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * VisitsController
+ *
+ * @description :: Server-side logic for managing visits
+ * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+ */
+
+module.exports = {
+
+  /**
+   * `VisitsController.create()`
+   */
+  create: function (req, res, next) {
+    var params = req.params.all();
+    console.log("Visit.create params=", params);
+
+    Visit.create(params, function(error, newVisit) {
+      if (error) return next(error);
+
+      res.status(201);
+      res.json(newVisit);
+    });
+  },
+
+
+  /**
+   * `VisitsController.find()`
+   */
+  find: function (req, res, next) {
+    var id = req.param('id');
+
+    if (id) {
```

Comment:
Czy id może być liczbą zero? Głównie uczulam na warunek z liczbą, ale też jest to potencjalny edge case.